patches/jsc_fix_arm64_jit_crash.patch

```diff
@@ -0,0 +1,41 @@
+diff -aur target-org/webkit/Source/JavaScriptCore/assembler/ARM64Assembler.h target/webkit/Source/JavaScriptCore/assembler/ARM64Assembler.h
+--- target-org/webkit/Source/JavaScriptCore/assembler/ARM64Assembler.h	2019-06-18 21:49:21.000000000 +0800
++++ target/webkit/Source/JavaScriptCore/assembler/ARM64Assembler.h	2019-06-19 15:14:46.000000000 +0800
+@@ -2863,7 +2863,36 @@
+
+     unsigned debugOffset() { return m_buffer.debugOffset(); }
+
+-#if OS(LINUX) && COMPILER(GCC_COMPATIBLE)
++#if defined(CUSTOMIZE_REACT_NATIVE)
++    static inline void linuxPageFlush(uintptr_t start, uintptr_t end)
```

Comment:
Also, I'd feel much safer if we only do that for __aarch64__, my understanding is that the errata only applies in that case. Perhaps this file does not get compiled on other CPU architectures but it wouldn't hurt to add that check here

Reply:
Make sense to me, will update soon.

Reply:
Updated and mark this conversation resolved.